frontend/public/components/storage/attach-storage.scss

```diff
@@ -0,0 +1,8 @@
+.co-storage {
+  &-radio {
+    margin-bottom: var(--pf-global--spacer--lg);
+    &__item {
+      margin-bottom: var(--pf-global--spacer--sm);
+    }
```

Comment:
Why is this required? Is the default PF wrong? If we disagree with it then we should have a bug raised on PF. But otherwise we should be using their default spacing.

Reply:
There is no spacing between radio items. 
Before:
![Screenshot from 2021-04-12 17-03-57](https://user-images.githubusercontent.com/54092533/114388301-2065ef00-9bb1-11eb-94b9-b4f852483f56.png)
After:
![Screenshot from 2021-04-12 17-04-07](https://user-images.githubusercontent.com/54092533/114388317-26f46680-9bb1-11eb-81eb-6200aa411b16.png)

Reply:
This seems to be the design intent from patternfly (https://patternfly-react.surge.sh/components/radio):
![image](https://user-images.githubusercontent.com/14068621/114464415-bbf85f00-9bb3-11eb-8074-05385795391c.png)

Whether we agree or not, we should be using PF styles. As I said, perhaps this is something we can bring up with PF to add spacing between adjacent radios.